quickemu

```diff
@@ -623,7 +623,7 @@ function configure_storage() {
         esac
 
         # https://blog.programster.org/qcow2-performance
-        if ! ${QEMU_IMG} create -q -f "${disk_format}" -o lazy_refcounts=on,preallocation="${preallocation}" "${disk_img}" "${disk_size}"; then
+        if ! ${QEMU_IMG} create -q -f "${disk_format}" -o lazy_refcounts=on,preallocation="${preallocation}",nocow=on "${disk_img}" "${disk_size}"; then
```

Comment:
```suggestion
local qemu_img_params
        if [[ "${disk_format}" == "qcow2" ]; then
            qemu_img_params="lazy_refcounts=on,preallocation=${preallocation},nocow=on"
        else
            qemu_img_params="preallocation=${preallocation}"
        fi
        if ! ${QEMU_IMG} create -q -f "${disk_format}" -o "${qemu_img_params}" "${disk_img}" "${disk_size}"; then
```
lazy_refcounts and nocow are only options for qcow2 disks, and quickemu allows for disk format to be specified. We need to make sure that qemu-img is not passed invalid parameters for the format at hand.